fix: all contributors summary not shown #272

iliassjabali opened the pull request:
## Checklist

- [x] Follow the [contributor guide](./CONTRIBUTING.md)
- [x] Update `./docs`
- [x] Recognize contributors with "@all-contributors please add <person> for <code, bug, docs, etc>"
- [x] Linked issues #272  and #62
<img width="995" height="849" alt="Screenshot 2025-11-18 at 8 34 13 PM" src="https://github.com/user-attachments/assets/ed6b8711-ea1a-461a-b9a8-95dd0134411f" />

iliassjabali committed:
feat: add ContributorsGrid component and update contributors reference

iliassjabali committed:
Merge branch 'mckinsey:main' into main

iliassjabali commented:
@all-contributors please add @iliassjabali for doc, bug

iliassjabali committed:
removed the userEffect:
-No client-side JavaScript for data fetching
Faster page loads — data is already available
-Better SEO — content is rendered server-side
-Simpler code — no loading states or error handling in the component

iliassjabali committed:
fix styling

iliassjabali committed:
Merge branch 'main' into bug/all-contributors-summary-not-shown-272

havasik committed:
Merge branch 'main' into bug/all-contributors-summary-not-shown-272

iliassjabali committed:
Merge branch 'main' into bug/all-contributors-summary-not-shown-272

iliassjabali commented:
@havasik  I am unsure why the those checks keep failing; i will look at them in few hours

david-gombos commented:
> @havasik I am unsure why the those checks keep failing; i will look at them in few hours

The PR title is an easy fix, just use `:` instad of `/`

iliassjabali committed:
Clean up comments in contributors.module.css

Removed unnecessary comments from contributors.module.css.

iliassjabali committed:
Merge branch 'main' into bug/all-contributors-summary-not-shown-272

iliassjabali committed:
chore: update testing setup with Vitest and add coverage configuration

- Changed the test script in package.json to use Vitest.
- Added a watch mode for tests.
- Introduced new devDependencies for testing: @testing-library/react, @vitejs/plugin-react, @vitest/coverage-v8, and jsdom.
- Created vitest.config.ts for test configuration and coverage reporting.

iliassjabali committed:
chore: exclude docs folder from codecov coverage

adriander committed:
Merge branch 'main' into bug/all-contributors-summary-not-shown-272
